WORKSPACE

```diff
@@ -1,5 +1,10 @@
 workspace(name = "tf_serving")
 
+local_repository(
+    name = "org_tensorflow",
```

Comment:
OK, auto-config and unbreaking us with TF@head is higher priority right now, so we can think about how to move this to workspace.bzl later. Thanks a lot for working on this!

When TF merges your change into master, please sync the TF submodule in this PR and we can run tests + merge.

The PR LGTM.